### DISCUSSION TITLE
Are there any plans to work with Bun in the future?

### DISCUSSION THREAD

**Discussion by regardt-nel** [meteor/meteor#12812] [Feature Requests]
The speed of Bun is truly impressive.  I have no idea what this would involve, but would there be any way to leverage Bun to improve Meteor build times?

**Comment by StorytellerCZ** [comment]
Check out the [forums discussion](https://forums.meteor.com/t/bun-and-meteor/60483?u=storyteller) for more details.

In short, no determination has been made yet (and we got bigger issues to fix first). Given how Meteor works it is not choosing between Node or Bun. It is one or the other.
While the community is excited about the speed improvements many are worried about parallels to early Meteor and Yarn, so we are cautious.

That said personally I had already swapped Node for Bun in the tool bundler, but didn't get much further than that.
